src/trigger_test_instance.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -euo pipefail
+repo=$1
+
+git config --global user.email "circleci-bot@greenpeace.org"
+git config --global user.name ":robot: CI Bot"
+git config --global push.default simple
+git config --global url."ssh://git@github.com".insteadOf "https://github.com" || true
+mkdir -p ~/.ssh/ && echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >~/.ssh/config
+
+instance_repo=planet4-test-$(cat /tmp/workspace/test-instance)
```

Comment:
Do we fail somewhere else before this if the file is empty or does not exist ?

Reply:
I moved this to be the first line so it makes the script fail if the file isn't there